### PR TITLE
Add Range::merge()

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
NB: Separate, largeish cleanup commit

**Comment by eile** [comment]
Writing Range unit test...

**Comment by eile** [comment]
done.

**Comment by tribal-tec** [comment]
+1

File: eq/frameData.cpp [hunk 1]
```diff
@@ -98,7 +98,7 @@ FrameData::~FrameData()
 {
     clear();
 
-    BOOST_FOREACH( Image* image, _impl->imageCache )
+    for( Image* image : _impl->imageCache )
```
Comment (bilgili) [inline eq/frameData.cpp]: c++11 already for Eq ?

Comment (eile) [inline eq/frameData.cpp]: Sure, anything above Brion should be fine. Brion to IO out from HPC machines.

**Pull request merged**